apps/src/p5lab/P5Lab.js

```diff
@@ -1151,7 +1151,7 @@ P5Lab.prototype.initInterpreter = function(attachDebugger = true) {
   this.JSInterpreter.parse({
     code,
     projectLibraries: this.level.projectLibraries,
-    blocks: this.isSpritelab ? [] : dropletConfig.blocks,
+    blocks: dropletConfig.blocks,
```

Comment:
[not blocking this PR] why did this break validation code?

Reply:
Because validation code often uses `console.log` and `console.log` was only getting populated into the interpreter state from Gamelab's droplet config. The change here makes it so that Spritelab will have access to gamelab's whole droplet config again, but that's just the fastest way to resolve this issue. Longer term, we need to figure out the right way to do this without the implicit dependence on Gamelab's droplet config.